### PR TITLE
Database errors

### PR DESCRIPTION
**Pull request by kgbfirehawk**
Added changes to App.py to take optional attributes in the future. Login API was not working as intended as well as a few User API requests. It is all working and as intended

### CODE REVIEW

**Review by lomaxanthony — APPROVED** [review]

**Review by lomaxanthony — APPROVED** [review]

File: backend/static/app.py [hunk 1]
```diff
@@ -84,13 +94,13 @@ def execute_sql_file(connection, sql_file_path):
...
-# GET, POST, PUT, and DELETE User. Also login via POST and change password via PUT #
-####################################################################################
+###################################
+# GET, POST, PUT, and DELETE User #
+###################################
 @app.route('/api/<username>', methods=['GET'])
 def get_user_info(username):
     """
-    Returns user_id, user_name, first_name, last_name, profile_pic_url, email, phone number, SMS notifications preference, email notifications preference, and preferred notification time
+    Returns user_id, user_name, email, phone number, SMS notifications preference, email notifications preference, and preferred notification time
```
Comment (robinxbaker) [inline backend/static/app.py:103]: Why were first_name, last_name, and profile_pic removed?

File: backend/static/app.py [hunk 2]
```diff
@@ -137,10 +144,6 @@ def add_user():
     User info is passed in body of message.
     Expected:
         user_name (string)
-        password (string)
```
Comment (robinxbaker) [inline backend/static/app.py:140]: Why was password removed?

File: backend/static/app.py [hunk 3]
```diff
@@ -799,6 +802,7 @@ def add_location():
             VALUES (%s)
         """
         cursor.execute(insert_query, (data['location_name'],))
+        cursor.execute(insert_query, (data['location_name'],))
```
Comment (robinxbaker) [inline backend/static/app.py:805]: Is this repeated for a reason (we have two of these statements now)

File: backend/static/app.py [hunk 4]
```diff
@@ -930,6 +927,7 @@ def add_recipe():
         insert_query = """
             INSERT INTO GroceryApp.Recipes (recipe_name, recipe_url, user_id, recipe_notification)
             VALUES (%s, %s, %s, %s)
+            VALUES (%s, %s, %s, %s)
```
Comment (robinxbaker) [inline backend/static/app.py:930]: Is this repeated for a reason (we have two of these statements now)

File: backend/static/app.py [hunk 5]
```diff
@@ -1054,6 +1052,7 @@ def add_ingredient():
         insert_query = """
             INSERT INTO GroceryApp.Ingredients (recipe_id, food_id, quantity_required)
             VALUES (%s, %s, %s)
+            VALUES (%s, %s, %s)
```
Comment (robinxbaker) [inline backend/static/app.py:1055]: Is this repeated for a reason (we have two of these statements now)

**Pull request closed without merging**